MessageBird/Net/RestClient.cs

```diff
@@ -35,15 +37,26 @@ public string UserAgent
             get { return string.Format("MessageBird/ApiClient/{0} DotNet/{1}", ApiVersion, ClientVersion); }
         }
 
-        public RestClient(string endpoint, string accessKey, IProxyConfigurationInjector proxyConfigurationInjector)
+        public RestClient(string endpoint, string accessKey, IProxyConfigurationInjector proxyConfigurationInjector, TimeSpan requestTimeout)
         {
             Endpoint = endpoint;
             AccessKey = accessKey;
             ProxyConfigurationInjector = proxyConfigurationInjector;
+            RequestTimeout = requestTimeout;
+        }
+
+        public RestClient(string endpoint, string accessKey, IProxyConfigurationInjector proxyConfigurationInjector)
+            : this(endpoint, accessKey, proxyConfigurationInjector, TimeSpan.FromSeconds(60))
+        {
+        }
+
+        public RestClient(string accessKey, IProxyConfigurationInjector proxyConfigurationInjector, TimeSpan requestTimeout)
+            : this(HttpsRestMessagebirdComEndpoint, accessKey, proxyConfigurationInjector, requestTimeout)
+        {
         }
 
         public RestClient(string accessKey, IProxyConfigurationInjector proxyConfigurationInjector)
-            : this(HttpsRestMessagebirdComEndpoint, accessKey, proxyConfigurationInjector)
+            : this(accessKey, proxyConfigurationInjector, TimeSpan.FromSeconds(60))
```

Comment:
I think it would be great to have a `TimeSpan DefaultTimeout` field to use here - then we can reuse that below when actually setting the timeout in `PrepareRequest`. What do you think?